### PR TITLE
Only take 1 value when using with ServerHistory

### PR DESCRIPTION
**Pull request by TylorS**
Only push a single value to the application when using ServerHistory. When using this for server-side rendering only 1 value will be pushed to the application.

Currently the stream returned is infinite, which works client-side, but leads to hacks needed to be placed server-side. This makes sure that on the server things are make finite, and can properly render html.

### CODE REVIEW

**Comment by TylorS** [comment]
cc @michalvankodev

**Comment by michalvankodev** [comment]
Yea this is a one solution but perhaps we could expose a method on the ServerHistory to complete the history$. So perhaps someone could push multiple Locations to the app to perhaps create some different type of state in his app.

File: src/makeHistoryDriver.ts [hunk 1]
```diff
@@ -2,6 +2,7 @@ import {StreamAdapter} from '@cycle/base';
 import {History, Location, HistoryDriverOptions, Pathname} from './interfaces';
 
 import {captureClicks} from './captureClicks';
+import {ServerHistory} from './serverHistory';
```
Comment (michalvankodev) [inline src/makeHistoryDriver.ts]: Not needed anymore.

**Comment by michalvankodev** [comment]
I just want to point out that this PR is making a server rendereding a breeze. Even http requests are handled on the server even in deeply nested routes.

**Comment by TylorS** [comment]
Awesome that's great to hear :+1:

**Comment by TylorS** [comment]
@michalvankodev Any further issues or concerns with this PR?

**Comment by michalvankodev** [comment]
@TylorS No. :) I'm still curious why it is not released ;)

**Comment by TylorS** [comment]
Should be good with v3.1.0 :+1:

**Pull request merged**